server/src/main.rs

```diff
@@ -6,16 +6,350 @@
     unused_import_braces,
     unused_qualifications,
     unsafe_code,
-    dead_code,
-    unused_results,
```

Comment:
If nab's is saying that we should keep this as compilation errors, then I agree.